<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyTextEdit</class>
 <widget class="QWidget" name="MyTextEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>573</width>
    <height>616</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="operationWidget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QFontComboBox" name="fontComboBox"/>
      </item>
      <item>
       <widget class="QComboBox" name="fontSizeComboBox"/>
      </item>
      <item>
       <widget class="QToolButton" name="smallToolButton">
        <property name="text">
         <string>S</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="largeToolButton">
        <property name="text">
         <string>L</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="colorToolButton">
        <property name="text">
         <string>C</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="italyToolButton">
        <property name="text">
         <string>I</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="boldToolButton">
        <property name="text">
         <string>B</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="underLineToolButton">
        <property name="text">
         <string>U</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="deleteLineToolButton">
        <property name="text">
         <string>D</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="textEdit"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
